.travis.yml

```diff
@@ -3,29 +3,48 @@ language: php
 matrix:
   fast_finish: true
   include:
-    - php: 5.5
+    - php: 5.6
       env:
         - STORAGE=doctrine
-    - php: 5.5
-      env:
-        - STORAGE=array
+        - COMPOSER_FLAGS="--prefer-lowest --prefer-dist --no-interaction"
     - php: 5.6
       env:
         - STORAGE=doctrine
-        - CODE_COVERAGE=true
     - php: 5.6
       env:
         - STORAGE=array
     - php: 7.0
       env:
         - STORAGE=doctrine
+        - COMPOSER_FLAGS="--prefer-lowest --prefer-dist --no-interaction"
+    - php: 7.0
+      env:
+        - STORAGE=doctrine
+    - php: 7.0
+      env:
+        - STORAGE=array
+    - php: 7.0
+      env:
+        - STORAGE=doctrine
+        - COMPOSER_FLAGS="--prefer-lowest --prefer-dist --no-interaction"
     - php: 7.0
+      env:
+        - STORAGE=doctrine
+    - php: 7.1
       env:
         - STORAGE=array
+    - php: 7.1
+      env:
+        - STORAGE=doctrine
+        - COMPOSER_FLAGS="--prefer-lowest --prefer-dist --no-interaction"
+    - php: 7.1
```

Comment:
we should test on 7.3

Reply:
cannot :) we would have to update phpunit and then i cannot test 5.6 😂

Reply:
Other bundles are also tested from 5.5 to 7.3 so this should be no problem when allowing newer phpunit version:

```
"phpunit/phpunit": "^4.8 || ^5.0",
```

Will fix this for you.